pig/src/main/java/org/jboss/pnc/bacon/pig/impl/pnc/PncBuilder.java

```diff
@@ -112,7 +114,10 @@ boolean isSuccessfullyFinished(String groupBuildId) {
                 case SUCCESS:
                     return verifyAllBuildsInGroupBuildInFinalStateWithProperCount(groupBuildId);
                 default:
-                    throw new RuntimeException("Build group failed " + groupBuild);
+                    throw new FatalException(
+                            "Build group failed {}/pnc-web/#/group-builds/{}",
+                            Config.instance().getActiveProfile().getPnc().getUrl(),
```

Comment:
Do you need `http://` on it?
Should this have a test?

Reply:
http:// is part of the pnc url so not needed here, as for test I don't think so, if url isn't correct it fails earlier in gathering of configuration part and this is just change of the printed info.
![Screenshot from 2021-03-30 14-04-46](https://user-images.githubusercontent.com/5735284/112985987-37442480-9161-11eb-8639-bfb71abc2fb5.png)

Reply:
Ok cool